docs/CLI.md

```diff
@@ -0,0 +1,142 @@
+# covalent cli
+
+## Overview
+
+CLI (Command Line Interface) for Covalent.
+
+
+### Usage
+
+To run the covalent tools cli do the following:
+
+* Clone this repository
+* npm i
+* npm run build
+* npm install -g
+
+#### The following commands are available in the CLI
+
+#### Help Usage
+```
+cov -h
+  Usage: cov [options]
+
+
+  Options:
+
+    -V, --version           output the version number
```

Comment:
i'm not sure that `-V` and `-v` are the best UX.  It's a bit confusing, and the defacto standard is small v (`-v`) for version.  Maybe have `-v` as version and remove `-V` altogether and just have the long form flag available.

Reply:
Or keep all versions and remove the singular version.